deploy/docker/owui-patches/tenant_role_from_db.py

```
@@ -0,0 +1,65 @@
+# neither the owui_role claim nor user_metadata (both confirmed empty
+# above), so tenant_id would be equally unreachable via that path.
+# Given that, this only ever resolves a role when the email has EXACTLY
+# ONE active tenant membership (the actual shape of the incident: a
+# brand-new self-serve tenant's sole OWNER). Zero or more than one
+# active membership is genuinely ambiguous and must never resolve
+# toward more privilege, so both fail closed to the existing fallback
+# (DEFAULT_USER_ROLE, "pending") rather than guessing.
+try:
+    import os as hive_os
+
+    hive_email = user.email if user else user_data.get('email')
```

Ruff (0.15.21)

[error] Undefined name `user`

(F821)